src/ansys_sphinx_theme/search/fuse_search.py

```diff
@@ -27,11 +27,24 @@
 
 from docutils import nodes
 
+PARAGRAPHS = [nodes.paragraph]
+TITLE = [nodes.title]
+LITERAL = [nodes.literal]
+ALL_NODES = [nodes.Text]
+ALL_NODES_WITHOUT_RAW = [  # type: ignore
+    nodes.paragraph,
+    nodes.title,
+    nodes.list_item,
+    nodes.field_list,
+    nodes.compound,
+    nodes.block_quote,
+]
```

Comment:
This is strange. The `ALL_NODES` contains only one element whereas the `ALL_NODES_WITHOUT_RAW` has even more. Why is that?

Reply:
I was just testing some node parameters. Yes, the 'all node' refers to all the raw text inside the docs, and it also includes the JS, as I mentioned privately. This was just for testing purposes. I'll clean it up a bit and incorporate your comments as well.